src/main/java/me/fromgate/reactions/logic/actions/ActionPotion.java

```diff
@@ -64,13 +68,28 @@ private String potionEffect(Player p, Parameters params) {
             amplifier = Math.max(params.getInteger("level", 1) - 1, 0);
             ambient = params.getBoolean("ambient", false);
         }
-        PotionEffectType pef = PotionEffectType.getByName(peffstr);
-        if (pef == null) return "";
+        PotionEffectType pef = getPotionEffectType(peffstr);
+
         PotionEffect pe = new PotionEffect(pef, duration, amplifier, ambient);
```

Comment:
Actually, it will throw npe if user will use invalid effect. Some null check is required.